funfact/lang/_tsrex.py

```diff
@@ -244,6 +237,7 @@ def __getitem__(self, indices):
 
 
 class EinopEx(TsrEx):
+    # override the `>>` behavior from TranspositionMixin
```

Comment:
Do we need `>>` in both places? Is the value of the `TranspositionMixin` class to be able to permute a single tensor instead of an expression?

Reply:
We do need both `__rshift__`, because the default one creates a new `tran` node, while the one here only modified the `outidx` field of the `ein` node.